src/Service/Payment.php

```diff
@@ -207,7 +207,11 @@ public function doPatchPayPalOrder(
 
         // Update Order
         try {
-            $orderService->updateOrder($checkoutOrderId, $request);
+            $orderService->updateOrder(
+                $checkoutOrderId,
+                $request,
+                Constants::PAYPAL_PARTNER_ATTRIBUTION_ID_PPCP
```

Comment:
in my source code source/vendor/oxid-solution-catalysts/paypal-client/generated/Service/Orders.php:117 only has 2 parameters the Constants::PAYPAL_PARTNER_ATTRIBUTION_ID_PPCP is ignored here

Reply:
with composer update it worked